packages/grafana-data/src/types/data.ts

```diff
@@ -28,6 +30,9 @@ export interface QueryResultMeta {
   /** Used to track transformation ids that where part of the processing */
   transformations?: string[];
 
+  /** Currently used to show results in Explore only in preferred visualisation option */
+  preferredVisualisationType?: PreferredVisualisationType;
```

Comment:
@ryantxu just a heads up we are adding this to query meta. It's nice solution for data sources that return different results sets (data frames) for the same query to direct explore to visualize them in the correct display (time series to graph and the instant query to table). And similar case for elasticsearch logs where one query returns time series and logs result in two different data frames. 

This now finally makes explore show time series in the table as well for other data sources that only return time series (removing the filter from the explore table only made it show non-time series).